### DISCUSSION TITLE
One ESP32, but multiple devices in HA

### DISCUSSION THREAD

**Discussion by alexanderniebuhr** [omerfaruk-aran/esphome_samsung_hvac_bus#131] [Questions]
Is there a way to only have one ESP32 in the setup, but have multiple indoor units all as separate devices in HA. I feel having all grouped under the same device, makes the experience less optimal. Also I don't know how Voice would work?

![image](https://github.com/lanwin/esphome_samsung_ac/assets/45965090/5902aa35-481c-4402-9c9a-d1edb930afb3)
![image](https://github.com/lanwin/esphome_samsung_ac/assets/45965090/27187cf8-40ec-446b-b547-7d714c374cc0)

**Comment by bzumik1** [comment]
Not possible at the moment, check the issue I raised few months ago. There is some work going on in ESPHome project which should allow it but currently it is not possible.

**Comment by lanwin** [comment]
As @bzumik1 already said. As far as I know this is currently a limitation of ESPHome.

As I understand it https://github.com/esphome/feature-requests/issues/1714 .. the ESP device should be more like an Climate Controller. 

In the end this should not mean anything. Most of the time you control entities in HA and not devices.
